MATH2269_final_project.Rmd

```diff
@@ -614,32 +616,34 @@ subsampled_data$pm10 <- subsampled_data$pm10 + 0.01
 ### Possible target variables
 
 `pm10` is target variable
-`yn80` possible target variable
+`yn80` possible target variable - *No, we have decided on pm10 only.*
 
 ```{r Mathematical model}
 
-source(here('src', 'model', 'model.R'))
-
-data_cleaned <- model_math(data_cleaned)
+# source(here('src', 'model', 'model.R'))
```

Comment:
You included this function originally? This is the section for that model diagram you did for the video presentation. It needs to go here too. So if you did it via an R Script then generate it here. Otherwise if you did a bit of post work in libra office just save that as an image and drop it in our src folder (or wherever is relevant) and we can import image using r markdown.